src/app/UrhoApp.cxx

```diff
@@ -90,44 +97,11 @@ void UrhoApp::create_viewport() {
 }
 
 void UrhoApp::setup_attachments(BoardData& board, const smce::VehicleConfig& vconf) {
```

Comment:
maybe you're still working on it but its missing an implementation

Reply:
Actually supposed to be deleted, thanks for the heads-up (+method got moved to EmuGlue)